### ISSUE TITLE
Please remove the ligature 0x

### ISSUE THREAD

**Issue by CodeCrazy-ywt** [JetBrains/JetBrainsMono#346]
Let's return to the origin: Why do we need ligatures? Take ≤ and ≥ for example. As we all know, these two symbols represents 'less-equal' and 'greater-equal'. However, they cannot be directly typed with a keyboard, so we came up with an alternative solution -- use a combination of symbols -- `<=` and `>=`. However, when we see these two symbol combinations, our brains needs some time to associate them with 'less-equal' and 'greater-equal', because when we were very little, it's ≤ and ≥ , not <= and >= ,that our teachers taught us to represent these two meanings. So the original purpose of ligatures is to reduce the workload of our brains and make the code more readable.
However, when it comes to `0x`, situations are different. Personally, I don't think it a good idea to turn 0x into 0×, because when we see `0×ff`, we tend to think it as '0 multiplied by `ff`', instead of a hexadecimal number that equals the decimal number 255, which runs counter to the original purpose of ligatures. Instead, I think removing the ligature is better.
In addition, I disagree with the idea of turning `0x` into `hex` or so on, because it changes `0x` into a completely different symbol (see `0xff`, `hexff` and `0₁₆ff`).
In summary, I think that removing the ligature for `0x` is a good idea.

**Comment by philippnurullin** [comment]
Hi @CodeCrazy-ywt All the discussion happen to be in this issue https://github.com/JetBrains/JetBrainsMono/issues/98
In the next update this ligature will be removed.

**Comment by CodeCrazy-ywt** [comment]
OK. Great thanks!

**Comment by philippnurullin** [comment]
Forgot to mention this in the Changelog. The ligature was removed in 2.210

**Comment by manide** [comment]
Hmm, using version 2.210, but still get a ligature for 0x when it's followed by a digit (nor for other characters though).
<img width="53" alt="0xLigature" src="https://user-images.githubusercontent.com/1315212/98803750-850bfa80-2415-11eb-82d1-4e14517b258d.png">

**Comment by philippnurullin** [comment]
Sorry, didn't remove all the code in the 2.210. You can grab the files from latest push (2.211). It's fixed there.

**Comment by CodeCrazy-ywt** [comment]
So I wonder when will v2.211 be released.

**Comment by Calinou** [comment]
@CodeCrazy-ywt It's released now: https://github.com/JetBrains/JetBrainsMono/releases/tag/v2.221
